node/actors/bft/src/chonky_bft/misc.rs

```diff
@@ -0,0 +1,105 @@
+use super::StateMachine;
```

Comment:
"misc" doesn't mean anything. If you cannot find a good place for this logic, then just move it to the parent module.

Reply:
I only wanted to have start and run on the parent module... What if I rename it to utils? I also accept other names.

Reply:
"utils" is as bad as "misc", it doesn't say anything. If you don't want to figure out a better layout, I'll take care of it later.

Reply:
Done.